components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -620,27 +620,18 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
 
         await this.guardAdminAccess("adminBlockUser", { req }, Permission.ADMIN_USERS);
 
-        let targetUser;
-        try {
-            targetUser = await this.userService.blockUser(req.id, req.blocked);
-        } catch (error) {
-            throw new ResponseError(ErrorCodes.NOT_FOUND, "not found");
-        }
+        const targetUser = await this.userService.blockUser(req.id, req.blocked);
```

Comment:
No need to wrap in try, as we throw an API ResponseError from the `userService.blockUser` call already.